### ISSUE TITLE
[BUG] users_info miss username parameter

### ISSUE THREAD

**Issue by monspid** [jadolg/rocketchat_API#9]
### Now fuction looks like this:
```
    def users_info(self, user_id, **kwargs):
        """Gets a user’s information, limited to the caller’s permissions."""
        return self.__call_api_get('users.info', userId=user_id, kwargs=kwargs)
```
### How to FIX:
```
    def users_info(self, user_id=None, username=None, **kwargs):
        """Gets a user’s information, limited to the caller’s permissions."""
        if user_id:
            return self.__call_api_get('users.info', userId=user_id, kwargs=kwargs)
        elif username:
            return self.__call_api_get('users.info', username=username, kwargs=kwargs)
        else:
            raise RocketMissingParamException('userID or username required')
```

**Comment by jadolg** [comment]
Would you PR your changes please? Also modifying the test would be awesome too ;)

**Comment by monspid** [comment]
It would be great if you push it as soon as possible to official repo, because it blocks me in my job.

**Comment by jadolg** [comment]
merged and pushed to PyPi version 0.6.3. Nice Work! :+1:
